python/lsst/meas/astrom/astrom.py

```diff
@@ -254,11 +259,11 @@ def useKnownWcs(self, sources, wcs=None, exposure=None, filterName=None, imageSi
         probably also want to turn OFF 'calculateSip'.
 
         This involves searching for reference sources within the WCS
-        area, and matching them to the given 'sources'.  If
+        area, and matching them to the given 'sourceCat'.  If
```

Comment:
I doubt it. It took me several hours to squash my commits since I kept running into problems, but when I was done I discarded the old repo with 27 commits. Technically it's probably still on github but I really don't want to try that again. (As to sources: this is a pet peeve of mine; I find the name entirely unhelpful. It is a collection of some kind, but what kind? A catalog and a list have entirely different behavior. I felt a bit guilty switching, but I'd really like to see us switch everywhere. Names like sources make life very hard for newbies and people who don't touch this code often. Also collections named with a plural violate our coding standards, or at least did at one time.)